.travis.yml

```diff
@@ -111,6 +111,21 @@ before_script:
   - echo -e "[pdns-sqlite3-1]\nDriver = SQLite3\nDatabase = ${PWD}/regression-tests/pdns.sqlite3\n\n[pdns-sqlite3-2]\nDriver = SQLite3\nDatabase = ${PWD}/regression-tests/pdns.sqlite32\n" > ${HOME}/.odbc.ini
   - export GODBC_SQLITE3_DSN=pdns-sqlite3-1
 
+  # ldap-backend test setup
+  - sudo apt-get -qq --no-install-recommends install slapd ldap-utils
```

Comment:
could the spaces be replaced by new-lines? this makes .travis.yml better diffable.